unblob/models.py

```diff
@@ -190,7 +204,7 @@ def default(self, obj):
             try:
                 return obj.decode()
             except UnicodeDecodeError:
-                return str(obj)
+                return obj.decode("utf-8", errors="surrogateescape")
```

Comment:
`try`-`except` construct is no longer needed, a simple `obj.decode("utf-8", errors="surrogateescape")` will always succeed.